Detections/SigninLogs/DiscoveryAttemptWithAzureHound.yaml

```diff
@@ -0,0 +1,53 @@
+id: 25a3fd0b-8fc1-42b9-8eca-867477218aea
+name: Azure AD Discovery attempt with AzureHound
+description: |
+  'The AzureHound collector for BloodHound (https://bloodhound.readthedocs.io/en/latest/data-analysis/bloodhound-gui.html) requires authentication using Connect-AzureAD and Connect-AZAccount. This detection checks for successfull authentication to Azure Active Directory PowerShell and then within 5 minutes also to Microsoft Azure PowerShell to try and identify possible use of AzureHound.'
+severity: Low
+requiredDataConnectors:
+  - connectorId: SignInLogs
+    dataTypes:
+      - SignInLogs
+queryFrequency: 1h
+queryPeriod: 1h
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - Discovery
+relevantTechniques:
+  - T1A0007
```

Comment:
This is not a correct Technique ID, I think you want T1087 for Account Discovery - https://attack.mitre.org/techniques/T1087/

Reply:
Fixed